src/common/ThemeContext.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const themes = {
+  default: {
+    header: 'default',
+  },
+  inverted: {
+    header: 'inverted',
+  },
+  light: {
+    header: 'light',
+  },
```

Comment:
Why have the specific value for `header` here? Should the `Header` component know how to display itself depending just on the active theme?

Reply:
The idea here is that each theme can be composed of different colors for each component.